### DISCUSSION TITLE
完全一致検索

### DISCUSSION THREAD

**Discussion by nak435** [increments/qiita-discussions#872] [要望・アイデア・不具合]
いつもお世話になります。

![qitta2](https://github.com/increments/qiita-discussions/assets/42625437/3cb1ba66-6f01-4b7b-9cff-0ad75d855a86)

完全一致検索したつもりなんですが、検索結果を見ると、そうはなっていないと思われます。
ご確認お願いします。

![qiita](https://github.com/increments/qiita-discussions/assets/42625437/b62372d9-48a5-4f21-8cb8-2889e39f3bad)

**Comment by tanaka-asami415** [comment]
@nak435 
ご報告ありがとうございます！ご不便をおかけして申し訳ありません。

検索結果を確認したところ、`M[34` もヒットしていそうです。
一度調査いたします🙇
